[2025-10-28T23:43:57.598+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka manual__2025-10-28T23:43:56.335635+00:00 [queued]>
[2025-10-28T23:43:57.608+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka manual__2025-10-28T23:43:56.335635+00:00 [queued]>
[2025-10-28T23:43:57.609+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-28T23:43:57.625+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-28 23:43:56.335635+00:00
[2025-10-28T23:43:57.635+0000] {standard_task_runner.py:60} INFO - Started process 201 to run task
[2025-10-28T23:43:57.641+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'manual__2025-10-28T23:43:56.335635+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmpc_01b34a']
[2025-10-28T23:43:57.644+0000] {standard_task_runner.py:88} INFO - Job 68: Subtask ConsumKafka
[2025-10-28T23:43:57.711+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka manual__2025-10-28T23:43:56.335635+00:00 [running]> on host 9680281a20bc
[2025-10-28T23:43:57.822+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-28T23:43:56.335635+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-28T23:43:56.335635+00:00'
[2025-10-28T23:43:57.827+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:43:57.831+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.5', 9092)]>: Broker version identified as 2.6
[2025-10-28T23:43:57.832+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:43:57.834+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-28T23:43:57.834+0000] {dag1_kafka_processing.py:71} INFO - Consommateur Kafka créé pour le topic 'source'. En attente de messages...
[2025-10-28T23:43:57.838+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:43:57.839+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:43:57.839+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:43:57.943+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-28T23:43:57.944+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-28T23:43:57.945+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-28T23:43:57.946+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-28T23:43:57.947+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-28T23:43:57.948+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.5', 9092)]>: connecting to kafka:9092 [('172.18.0.5', 9092) IPv4]
[2025-10-28T23:43:57.949+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Connection complete.
[2025-10-28T23:43:58.050+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-28T23:43:58.054+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-585471a9-6f08-4f41-a715-ddb6e60585a4 for group ***-consumers-dag1; will retry join-group
[2025-10-28T23:43:58.058+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-28T23:43:58.059+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-28T23:44:01.066+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 3 (member_id: kafka-python-2.2.15-585471a9-6f08-4f41-a715-ddb6e60585a4, protocol: range)>
[2025-10-28T23:44:01.067+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-28T23:44:01.073+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-28T23:44:01.074+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-28T23:44:07.838+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-28T23:44:07.839+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-28T23:44:07.841+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-28T23:44:07.842+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:44:07.842+0000] {fetcher.py:786} INFO - Fetch to node 1 failed: Cancelled: <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>
[2025-10-28T23:44:07.843+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.5', 9092)]>: Closing connection. 
[2025-10-28T23:44:07.843+0000] {dag1_kafka_processing.py:77} WARNING - Aucun message reçu du topic 'source'. Le DAG s'arrêtera.
[2025-10-28T23:44:07.844+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-28T23:44:07.858+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251028T234356, start_date=20251028T234357, end_date=20251028T234407
[2025-10-28T23:44:07.901+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-28T23:44:07.935+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
